src/Model/Entity/Asset.php

```diff
@@ -131,8 +131,8 @@ public function isViewableInBrowser(): bool
     public function isImage(): bool
     {
         return $this->exists()
-            && Strings::before((string)$this->mimetype, '/') === 'image'
-            && !Strings::contains((string)Strings::after($this->mimetype, '/'), 'svg');
+            && Strings::before($this->mimetype ?? '', '/') === 'image'
+            && !Strings::contains((string)Strings::after($this->mimetype ?? '', '/'), 'svg');
```

Comment:
`$this->mimetype` is nullable but the `Strings` class requires the first param to only be a string, no null.